build.sbt

```diff
@@ -4,15 +4,15 @@ import uk.gov.hmrc.DefaultBuildSettings.itSettings
 val appName = "customs-email-frontend"
 
 val silencerVersion = "1.7.16"
-val scala3_3_3 = "3.3.3"
+val scala3_3_4 = "3.3.4"
 val bootstrap = "9.1.0"
```

Comment:
We're using bootstrap twice, here and in the project/AppDependencies.scala and we're updating one but not the other, which will cause some issues. I think it would be a good idea to keep them the same, or better yet remove one of them make both the build.sbt and the AppDep use the same val.